docs/index.asciidoc

```diff
@@ -33,6 +33,8 @@ https://github.com/elastic/logstash-forwarder[logstash-forwarder]
 tool that has been replaced by
 https://github.com/elastic/beats/tree/master/filebeat[Filebeat].
 
+NOTE: Consider using the {logstash-ref}/plugins-inputs-beats.html[beats input plugin] instead.  
+The Beats input implements the old and the new lumberjack protocol. 
```

Comment:
I would suggest capitalizing Beats and Lumberjack. Also, the mention of the old and new Lumberjack protocols might be confusing given that they are not discussed anywhere else. I'd suggest something along the lines of "the Beats input is actively developed and backward-compatible with the Lumberjack input."

Reply:
If we retain the phrase `the old and the new`, would it be appropriate to clarify the protocol versions?  Something like:
> old (v1) and the new (v2) lumberjack protocol.